docs/install.md

```diff
@@ -125,6 +125,13 @@ See [usage.md](usage.md) for usage instructions, such as how to update.
 
 Tutorial: install and configure NixOS for nix-bitcoin on your own hardware
 ---
+
+## 0. Preparation
+
+1. Optional: Disable Simultaneous Multi-Threading (SMT) in the BIOS
+
+    Researchers recommend disabling (SMT), also known as Hyper-Threading Technology in the Intel® world to significantly reduce the impact of speculative exacution-based attacks (https://mdsattacks.com/).
```

Comment:
Another mitigation would be installing intel microcode updates, possibly through: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/hardware/cpu/intel-microcode.nix